autopts/bot/common_features/mail.py

```diff
@@ -106,38 +106,49 @@ def profile_summary(tc_results):
     for tg in test_groups.values():
         tg.get_pass_rate()
 
-    table_rows = ""
-    for suite, stats in test_groups.items():
-        table_rows += f"""
-            <tr>
-                <td>{suite}</td>
-                <td>{stats.total}</td>
-                <td>{stats.passed}</td>
-                <td>{stats.failed}</td>
-                <td>{stats.pass_rate:.2f} %</td>
-            </tr>
+    if target == 'mail':
```

Comment:
Suggest to either define the targets as an string enum or something else, instead of relying on comparing literal values